doctl 1.0.0

phillbaker opened the pull request:
`brew install doctl`

### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

v1.0.0: https://github.com/digitalocean/doctl/releases/tag/v1.0.0

phillbaker commented:
The build failures look related to https://github.com/Homebrew/homebrew/pull/49279 and adb5395b35d65f3edce574fdfd163c51b22f8ad58dfc1db4832eb219670c8d9b (updates to building with go1.6), however, that's for the existing 0.0.16 version. The new beta does work, but doesn't look tested.

DomT4 commented on Library/Formula/doctl.rb:
```diff
@@ -3,8 +3,8 @@
   homepage "https://github.com/digitalocean/doctl"
-  url "https://github.com/digitalocean/doctl/archive/0.0.16.tar.gz"
-  sha256 "4f4805f36fd0d437331c25a183471419d10a680721f7f5a890b4109319d605ed"
+  url "https://github.com/phillbaker/doctl/archive/0.0.17.tar.gz"
```
Is this a fork? The DO upstream doesn't seem to have moved anywhere.

phillbaker replied:
Whoops, thanks for catching that. I was testing and pushed a tag of existing code to my fork. I'll bug them to see if they can do the same.

ilovezfs commented:
Test is failing for your 0.0.17 tag because there's some bug that expects DIGITALOCEAN_API_KEY to be set even if you're just running help. Also -v seems to be broken in 0.0.17. Both bugs do not appear to affect the current stable or the devel version. The test also looks like it's going to need updating because the semantics for help changed between the versions.

phillbaker commented:
An official v1.0.0 was released upstream! Updated the commits to point to that tarball and simplified the formula to match the new version. Another look would be appreciated!

dunn commented:
Thank you! Merged as a2da4fd3dd748a59a6efff52a5bf949a643c69a4.
